crates/shared/src/bad_token/token_owner_finder/blockscout.rs

```diff
@@ -26,6 +26,7 @@ impl BlockscoutTokenOwnerFinder {
             Chain::Sepolia => "https://eth-sepolia.blockscout.com/api",
             Chain::ArbitrumOne => "https://arbitrum.blockscout.com/api",
             Chain::Base => "https://base.blockscout.com/api",
+            Chain::Hardhat => return Err(anyhow::anyhow!("Hardhat chain not supported")),
```

Comment:
As we are using anyhow, let's use dedicated macro for early return:
`anyhow::bail!("Hardhat chain not supported")`.